File: Dart_Source_Files/payload/Answer.dart
// ignore_for_file: file_names

class Answer {
  String answer;
  String correct;

  Answer({required this.answer, required this.correct});

  factory Answer.fromJson(Map<String, dynamic> json) {
    return Answer(
        answer: json["answer"], correct: (json["correct"]) ? "true" : "false");
  }
}

File: src/main/resources/static/my_scripts/payload/Answer.js
class Answer {  

    /**
     * 
     * @param {string} answer 
     * @param {boolean} correct 
     */
    constructor(answer, correct) {

        this.answer = answer;
        this.correct = correct;
    }

    /**
     * 
     * @param {object} json 
     * @returns {Answer}
     */
    static fromJson(json) {

        return new Answer(json.answer, json.correct);
    }
}

export {Answer};